ccdaservice/package-lock.json

```diff
@@ -2601,7 +2601,6 @@
             "integrity": "sha512-3wdGidZyq5PB084XLES5TpOSRA3wjXAlIWMhum2kRcv/41Sn2emQ0dycQW4uZXLejwKvg6EsvbdlVL+FYEct7A=="
         },
         "oe-blue-button-generate": {
-	    "name": "oe-blue-button-generate",
```

Comment:
Can you back out this change?  I'm assuming this got checked in by accident.

Reply:
hi @adunsulag, it's okay since is already in master from https://github.com/openemr/openemr/pull/7850/commits/c974843bcc9deed86b70cf6cf6154f35e1ef26f7